### ISSUE TITLE
Accessibility: column headers and query results are rendered as separate tables instead of a single one

### ISSUE THREAD

**Issue by Neurrone** [mtxr/vscode-sqltools#471]
**Describe the bug**
Column headers and result rows are rendered as separate tables instead of a single one.

**To Reproduce**
1. Run a query and view results.
2. This will be noticeable with a screen reader. I happen to use NVDA, which detects these as 2 separate tables.

**Expected behavior**
Column headers and results should be rendered as a single table. This improves accessibility for screen reader users. If this were in a single table, the screen reader would announce column headers (`<th>`) correctly when moving across tables, equivalent to visually scanning upwards, which makes understanding large tables much easier.

**Desktop (please complete the following information):**
 - SQLTools Version: v0.21
 - VSCode Version: 1.42.0 insider
 - Driver Used/Version: N/A
 - OS: Windows

**Additional context**

**Comment by mtxr** [comment]
That's currently not possible. The library used to render the table doesn't support this.

I understand the need of this change, but there are no plans to move to a new library since this is one of the most used around the web.

Would you mind opening an issue for the library itself? It's @devexpres/dx-react-grid.

**Comment by Neurrone** [comment]
@mtxr I just tried looking at the [material UI example](https://codesandbox.io/s/13qvz1qqzl), which does render tables correctly. Could you provide more info on exactly how the library is being used, or if you wouldn't mind, it might be better if you could open the issue, if that's not easy to explain to someone who hasn't used that library before.

**Comment by mtxr** [comment]
This is the example that fits better what we are doing: https://devexpress.github.io/devextreme-reactive/react/grid/docs/guides/lazy-loading/

I'm implementing lazy loading, but the table already works like that to have the fixed header in the top.

I'll open an issue, but I don't believe it's an upcoming change, ok? It's pretty hard to have that behavior without having 2 tables

**Comment by Neurrone** [comment]
@mtxr understood, thanks!
